calico/networking/ip-autodetection.md

```diff
@@ -109,28 +175,38 @@ Where autodetection methods are based on:
 
 - **Including matching interfaces**
 
-  A regular expression in golang syntax that includes interfaces that match. For example:
+  A regular expression in Golang syntax that includes interfaces that match. For example:
 
   ```
   kubectl set env daemonset/calico-node -n kube-system IP_AUTODETECTION_METHOD=interface=eth.*
   ```
 
 - **Excluding matching interfaces**
 
-  A regular expression in golang syntax that excludes interfaces that match. For example:
+  A regular expression in Golang syntax that excludes interfaces that match. For example:
 
   ```
   kubectl set env daemonset/calico-node -n kube-system IP_AUTODETECTION_METHOD=skip-interface=eth.*
   ```
   
 - **Kubernetes Node IP**
 
-  An IP address assigned to kubernetes node (INTERNAL-IP)
+  An internal IP address assigned to the Kubernetes node.
 
   ```
   kubectl set env daemonset/calico-node -n kube-system IP_AUTODETECTION_METHOD=kubernetes-internal-ip
   ```
 
+- **Including CIDRs**
+
+  A list of IP ranges in CIDR format to determine valid IP addresses on the node to choose from. For example:
```

Comment:
"Calico will select any IP address from the node that falls within the given CIDRs"